contracts/src/resolver/OwnedResolver.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import {AddrResolver} from "@ens/contracts/resolvers/profiles/AddrResolver.sol";
+import {ABIResolver} from "@ens/contracts/resolvers/profiles/ABIResolver.sol";
+import {ContentHashResolver} from "@ens/contracts/resolvers/profiles/ContentHashResolver.sol";
+import {DNSResolver} from "@ens/contracts/resolvers/profiles/DNSResolver.sol";
+import {InterfaceResolver} from "@ens/contracts/resolvers/profiles/InterfaceResolver.sol";
+import {NameResolver} from "@ens/contracts/resolvers/profiles/NameResolver.sol";
+import {PubkeyResolver} from "@ens/contracts/resolvers/profiles/PubkeyResolver.sol";
+import {TextResolver} from "@ens/contracts/resolvers/profiles/TextResolver.sol";
+import {ExtendedResolver} from "@ens/contracts/resolvers/profiles/ExtendedResolver.sol";
+
+/**
+ * @title OwnedResolver
+ * @dev A simple resolver anyone can use; only allows the owner of a node to set its
+ * address.
+ */
+contract OwnedResolver is
+    OwnableUpgradeable,
+    ABIResolver,
+    AddrResolver,
+    ContentHashResolver,
+    DNSResolver,
+    InterfaceResolver,
+    NameResolver,
+    PubkeyResolver,
+    TextResolver,
+    ExtendedResolver
+    {
+
+    function initialize(address _owner) public initializer {
+        __Ownable_init(_owner); // Initialize Ownable
```

Comment:
UUPS initialization must be added here as in [example implementation](https://github.com/ensdomains/verifiable-factory/blob/3293069b7309987e3118e5c6228e2a7ca6865721/README.md#example-implementation).